daffodil-runtime1/src/main/scala/org/apache/daffodil/api/DFDLParserUnparser.scala

```diff
@@ -263,9 +263,11 @@ object DFDL {
   /**
    * Used in SAXInfosetEvent.causeError to identify when an unexpected error occurs
    */
-  class DaffodilUnhandledSAXException(description: String, cause: Exception)
-    extends org.xml.sax.SAXException(description, cause)
 
+  class DaffodilUnhandledSAXException(description: String, cause: Exception)
+    extends org.xml.sax.SAXException(description, cause) {
+      def this(cause: Exception)= this(null, cause)
```

Comment:
A unit test (not a TDML test, just regular unit test) should construct one of these objects so this line is covered.